lib/prod-server.js

```diff
@@ -13,13 +15,19 @@ module.exports = ({
   port = 8080,
   staticDir,
   runner,
-  basePath
+  basePath,
+  assetPath
 }) => {
   const baseMatch = new RegExp(`^${basePath}`)
   server.use(logger({
     format: '[%date - %time] %status %method %url'
   }))
-  server.use(serveStatic(staticDir))
+  if (assetPath) {
+    const mountPoint = urllib.parse(assetPath).path
```

Comment:
So this is what allows the server to act as an origin if needs be? Nifty. Still would prefer `assetUrl` ... even in this case (which, granted, is a little weird).